docs/play/cdc-pay-integration.md

```diff
@@ -0,0 +1,151 @@
+            {
+                www.SetRequestHeader("Authorization", "Bearer YOUR_PUBLISHABLE_KEY");
+                yield return www.SendWebRequest();
+                Response data = JsonUtility.FromJson<Response>(System.Text.Encoding.UTF8.GetString(www.downloadHandler.data));
+                
+                if (www.result != UnityWebRequest.Result.Success)
+                {
+                    Debug.Log(www.error);
+                    Debug.Log(www.downloadHandler.data);
+                }
+                else
+                {
+                    webViewObject = (new GameObject("WebViewObject")).AddComponent<WebViewObject>();
+                    webViewObject.Init((msg) => {
+                    Debug.Log(msg);
+                    });
+                    webViewObject.LoadURL(data.payment_url);
+                    webViewObject.SetVisibility(true); 
+                }                    
+            }
+        }
+    }
+    ```
+
+5. Create an empty game object called script, and add the `CdcPay.cs` script to it.
+
+6. Drag the script gameobject into the button onClick component and select the `CdcPay()` method
+
+6. Run from the editor and click the checkout button.
+
+    Click Button 
+    <img src="./assets/web-view/web-view-integration-step-7a.png" />
+    Complete test Payment
+    <img src="./assets/web-view/web-view-integration-step-7b.png" />
+
+7. Complete a test order and retrieve the payload via webhook as explained in the next section.
+    <img src="./assets/web-view/web-view-integration-step-8.png" />
+::: tip
+Client Logic can be invoked in the game directly. Therefore, the approve function should only handle screen changes. It should be accompanied by server side logic like [Webhooks](https://pay-docs.crypto.com/?shell#api-reference-webhooks) to complete your orders.
+:::
+
+## Webhooks
+Webhooks make it easier to integrate with Crypto.com Pay by allowing you to subscribe to a set of events. You can refer to [Resources](https://pay-docs.crypto.com/?shell#api-reference-resources) for the schema of the objects.
+
+### Overview
+To simulate a purchase and retrieve the payload from the CDC Pay events, you can use this webhook server [example](https://github.com/cronos-labs/play-cpp-sdk/tree/main/demo).
+
+### Requirements
+- [node.js](https://nodejs.org/en/)) 
+- [ngrok](https://ngrok.com/)) 
+
+### Installation
+
+ 1. Clone the following repository
+    ```sh
+    git clone https://github.com/cronos-labs/play-cpp-sdk.git
+    ```
+2.  Run the following commands:
+    ```sh
+    cd demo
+    npm install
+    node server.js
+    ```
+    ::: tip
+    Please ensure to replace the `SIGNATURE_SECRET` from you merchants dashboard after adding the webhook endpoint.
+    :::
+4. Open a new terminal and run the following command:
+    ```sh
+    ngrok http 4567
+    ```
+5. Add the forwarding url from the ngrok GUI as your payload endpoint in your merchant dashboard.
+    <img src="./assets/webhook/webhook-step-1.png" />
```

Comment:
Is the signature_secret in imagea and publishable key in image 2 ok to be uploaded?